client_mapper.go

```diff
@@ -0,0 +1,26 @@
+package client
```

Comment:
How about we stick `client_mapper.go` and `unsafe_resource_interface.go` in their own `dynamic/` dir/pkg just to keep the top-level clean?

Reply:
Done.